source/pacmap/pacmap.py

```diff
@@ -869,9 +869,11 @@ def __init__(self,
             self.random_state = 0
             _RANDOM_STATE = None  # Reset random state
 
-        if self.n_components < 2:
+        if self.n_components < 1:
             raise ValueError(
                 "The number of projection dimensions must be at least 2.")
+        if self.n_components == 1:
```

Comment:
The warning should also be applied to all cases where `self.n_components != 2`. Similarly, the warning message should also mention that `Dimension != 2 case has not been thoroughly tested.`

Reply:
I think it may be a bit too verbose to warn for any choice of dimension > 2 from a software perspective. I did it as a debug statement for now - what do you think?